socialdistribution/settings.py

```diff
@@ -0,0 +1,83 @@
+"""
+Django settings for socialdistribution project.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.7/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.7/ref/settings/
+"""
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'k-1y7cv9s-8*x^s+kp@ylfj+*akf)&7by&43e%0160p25qy6c3'
+
```

Comment:
Okay, I've updated the gitignore, thanks for that pointer. As for the secret key and DB stuff. I'm not going to worry about it for now. We can do some fancier work surrounding it if we think it's necessary or Hindle makes a stink about it. I realize it's not a best practice, but it's kind of an unnecessary hassle at this point.